README.rst

```diff
@@ -243,6 +243,11 @@ The following `command line arguments`_ can also be used to modify the defaults:
                            Normalize string quotes or prefixes. This can be used
                            to override `skip_string_normalization = true` from a
                            configuration file.
+    --skip-magic-trailing-comma
```

Comment:
That will be handy if `--no-skip-magic-trailing-comma`, 
I think instead of keeping on adding `--no` we can allow the user to specify true/false, 

e.g

```
--magic-trailing-comma false/no
```

Reply:
> `--magic-trailing-comma false/no`

Yes, but then we'd need to change `--skip-string-normalization` and `--no-skip-string-normalization` as well for consistency.

Isn't it best to mirror the convention used in Black? They have `--skip-magic-trailing-comma` without a boolean value. Granted, they don't provide the `--no-*` variant, but we *could* do that as an enhancement.

On the other hand, as long as no-one has encountered a use case for `--no-skip-magic-trailing-comma`, I'm also ok with leaving it out for now.